presidio-anonymizer/app.py

```diff
@@ -25,9 +28,11 @@ def anonymize():
                 data = AnonymizerRequest(content)
                 text = AnonymizerEngine().anonymize(data)
             except InvalidParamException as e:
-                return e.err, 400
-            except Exception:
-                # TODO add logger 2652
+                logging.debug(
+                    {f"failed to anonymize text with validation error: {e.err_msg}"})
+                return e.err_msg, 400
```

Comment:
consider error code 422 Unprocessable Entity, since this is semantic error and not a general purpose "bad request"

Reply:
Very nice! thanks. Done.

Reply:
Not for all cases though.. for not passing a json at all or 'analyzer_results' at all, 422 might be misleading

Reply:
That's Avishays comment since the handling of an empty json is not in this section :)

Reply:
What about not passing analyzer_results?
Not critical since we are not 100% restful anyway, but might be confusing